src/eko/couplings.py

```diff
@@ -577,12 +644,244 @@ def rge(_t, a, beta_qcd_vec, beta_qed_vec):
             rge,
             (0, u),
             a_ref,
-            args=[beta_qcd_vec, beta_qed_vec],
+            args=[beta_qcd_vec, beta_qcd_mix, beta_qed_vec, beta_qed_mix],
             method="Radau",
             rtol=1e-6,
         )
         return np.array([res.y[0][-1], res.y[1][-1]])
 
+    def compute_exact_fixed_alphaem(self, a_ref, nf, scale_from, scale_to):
+        """Compute couplings via |RGE| with fixed alphaem.
+
+        Parameters
+        ----------
+        as_ref : numpy.ndarray
+            reference alpha_s and alpha
+        nf : int
+            value of nf for computing alpha_i
+        scale_from : float
+            reference scale
+        scale_to : float
+            target scale
+
+        Returns
+        -------
+        numpy.ndarray
+            couplings at target scale :math:`a(Q^2)`
+        """
+        u = np.log(scale_to / scale_from)
+
+        if self.order in [(1, 0), (1, 1)]:
+            return couplings_expanded_fixed_alphaem(
+                self.order, a_ref, nf, scale_from, float(scale_to)
+            )
+
+        beta_vec = [beta_qcd((2, 0), nf)]
+        # NLO
+        if self.order[0] >= 2:
+            beta_vec.append(beta_qcd((3, 0), nf))
+            # NNLO
+            if self.order[0] >= 3:
+                beta_vec.append(beta_qcd((4, 0), nf))
+                # N3LO
+                if self.order[0] >= 4:
+                    beta_vec.append(beta_qcd((5, 0), nf))
+        if self.order[1] >= 1:
+            beta_vec[0] += a_ref[1] * beta_qcd((2, 1), nf)
+        b_vec = [i / beta_vec[0] for i in beta_vec]
+        rge_qcd = self.unidimensional_exact(
+            beta_vec[0],
+            b_vec,
+            u,
+            a_ref[0],
+            method="Radau",
+            rtol=1e-6,
+        )
+        return np.array([rge_qcd, a_ref[1]])
+
+    # def compute_exact_decoupled_running(self, a_ref, nf, scale_qcd_from, scale_qed_from, scale_to):
```

Comment:
If you want to keep these 2 functions, add a comment explaining why they are commented

Reply:
In this case I was implementing the decoupled running (mostly for benchmarking against apfel) but I never did it to the end. Maybe we can just delete it.